pkg/api/config.go

```diff
@@ -68,21 +72,36 @@ type LogConfig struct {
 	Output string
 }
 
+type ExtensionConfig struct {
+	Search *SearchConfig
+}
+
+type SearchConfig struct {
+	// CVE search
+	CVE *CVEConfig
+}
+
+type CVEConfig struct {
+	UpdateInterval time.Duration // should be 2 hours or more, if not specified default be kept as 24 hours
```

Comment:
So the config param is called "UpdateInterval" not "PeriodicUpdates"? and it is a duration value, not bool.

Reply:
Yes

Reply:
Should it be a Duration? That's supposed to be nanoseconds, right? It looks like you're using it as an integer number of hours, should we just make it an int and avoid confusion here?

When you use it later you're multiplying by time.Hour in `time.Sleep(UpdateInterval * time.Hour)` anyway

Reply:
Ok I will make that as an integer and use it like that time.Sleep(time.Duration(c.Config.Extensions.Search.CVE.UpdateInterval) * time.Hour)

Reply:
No, leave this as time.Duration and in JSON config it should be "24h"
https://golang.org/pkg/time/#ParseDuration

If it is less that 2h, then default to 2h and throw a warning.

Reply:
> No, leave this as time.Duration and in JSON config it should be "24h"
> https://golang.org/pkg/time/#ParseDuration
> 
> If it is less that 2h, then default to 2h and throw a warning.

that makes sense too. didn't realize the json was supposed to be a string. @shimish2 sorry for the extra work there :)

Reply:
@shimish2 pls. test and verify just the same

Reply:
> > No, leave this as time.Duration and in JSON config it should be "24h"
> > https://golang.org/pkg/time/#ParseDuration
> > If it is less that 2h, then default to 2h and throw a warning.
> 
> that makes sense too. didn't realize the json was supposed to be a string. @shimish2 sorry for the extra work there :)

No problem @mikemccracken